kolibri/deployment/default/settings/base.py

```diff
@@ -190,6 +190,14 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+# File Storage Backend
+# https://docs.djangoproject.com/en/3.2/ref/files/storage/
+
+if not os.environ.get("DEFAULT_FILE_STORAGE"):
+    if conf.OPTIONS["FileStorage"]["STORAGE_BACKEND"] == "gcs":
+        DEFAULT_FILE_STORAGE = "kolibri.utils.file_storage.KolibriFileStorage"
+        BUCKET_NAME = os.getenv("GCS_BUCKET_NAME") or "kdp-csv-reporting-develop"
```

Comment:
This is borrowed from @jredrejo but should probably be set to a different fallback bucket.

We also may want to ensure that the GCS_BUCKET_NAME var is set to instance-specific buckets? 
cc @DXCanas

Reply:
We should have a BUCKET NAME option in our options.py as well - we definitely should not be hard coding anything specific about any platform in here.

If `GCS_BUCKET_NAME` is an environment variable that is set by default on GCP (but I don't think it is) we can specify specific env vars in the options configuration to draw values from.

Reply:
Rather than subclassing GoogleCloudStorage, we should just set the additional settings here for it.